core/server/monkey-patch.js

```diff
@@ -0,0 +1,28 @@
+var TimezonedDate = require('timezoned-date'),
+    lodash = require('lodash'),
+    clonedeep = require('lodash.clonedeep'),
+    toString = require('lodash.tostring');
+
+// ######## LODASH
+// timezoned-date does not work with lodash.cloneDeep 3.x
+lodash.cloneDeep = clonedeep;
+
+// @TODO: reason?
+lodash.toString = toString;
+
+// ######## NATIVE JS DATE
+// remember original timezone offset of server
+global.ServerTimezoneOffset = new Date().getTimezoneOffset();
+
+// timezoned-date is able to force UTC at process level
+// we only use it when local system is NOT UTC
+if (ServerTimezoneOffset !== 0) {
```

Comment:
This check isn't reliable. Imagine a server in the UK which is set up to adjust its clock for DST. Its time zone isn't UTC even though it looks like that if you check `new Date().getTimezoneOffset()` in winter. You'd better check two dates, like Jan 1 and Jun 1. It's not ultimately reliable, but still much better. See [this question](http://stackoverflow.com/questions/11887934/check-if-daylight-saving-time-is-in-effect-and-if-it-is-for-how-many-hours) on SO,